android/app/src/main/kotlin/io/ion/app/MainActivity.kt

```diff
@@ -1,6 +1,99 @@
 package io.ion.app
 
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import com.banuba.sdk.pe.PhotoCreationActivity
+import com.banuba.sdk.pe.BanubaPhotoEditor
+import com.banuba.sdk.pe.data.PhotoEditorConfig
 import io.flutter.embedding.android.FlutterActivity
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugins.GeneratedPluginRegistrant
+import java.io.File
 
-class MainActivity: FlutterActivity() {
+class MainActivity : FlutterActivity() {
```

Comment:
As I understand it, does this class not dispose of resources?

Reply:
no there is no any callback for it on documentation.